src/Stormpath/Oauth/CreateGrantAuthenticationAttempt.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+
+namespace Stormpath\Oauth;
+
+
+use Stormpath\Resource\AccountStore;
+use Stormpath\Resource\Resource;
+
+class CreateGrantAuthenticationAttempt extends Resource
```

Comment:
I think this should just be named `GrantAuthenticationAttempt`. Although, I'm not 100% sure of what this represents on the REST API.